tests/integration_testnet/api/rpc/accounts.test.js

```diff
@@ -127,7 +127,7 @@ describe('Method get.accounts', () => {
 		});
 
 		it('returns delegate data by delegate name', async () => {
-			const { result } = await getAccounts({ username: accounts.delegate.username });
+			const { result } = await getAccounts({ username: accounts.delegate.delegate.username });
```

Comment:
Why is that delegate object written twice? Did the API change?

Reply:
There was a bug in test. Its fixed now